packages/dids/src/methods/did-dht.ts

```diff
@@ -1188,13 +1193,14 @@ export class DidDhtDocument {
     didDocument.service?.forEach((service, index) => {
       const dnsRecordId = `s${index}`;
       serviceIds.push(dnsRecordId);
-      const serviceId = service.id.split('#').pop()!; // Remove fragment prefix, if any.
```

Comment:
so this should be returned

Reply:
@decentralgabe -- nothing changed in that line -- the `extractDidFragment()` does the exact same thing as `.split('#').pop()`